wandb/sdk/data_types/table_decorators.py

```diff
@@ -0,0 +1,100 @@
+"""Decorators for W&B Table operations."""
+
+from __future__ import annotations
+
+import sys
+from functools import wraps
+from typing import Any, Callable, TypeVar
+
+import wandb
+
+if sys.version_info < (3, 10):
+    from typing_extensions import Concatenate, ParamSpec
+else:
+    from typing import Concatenate, ParamSpec
```

Comment:
Can just use the `from typing_extensions` import without the `sys.version_info` check (I learned this recently myself)